lms/templates/courseware/course_updates.html

```diff
@@ -1,11 +1,9 @@
 <%! from django.utils.translation import ugettext as _ %>
-<section aria-labelledby="course-updates-heading">
```

Comment:
I spoke with @cptvitamin, this was removed on purpose as it was not needed and covered by existing visible text.